src/index.js

```diff
@@ -0,0 +1,53 @@
+import './style.css';
+
+const tasks = [
+  {
+    description: 'Exercise',
+    completed: 'true',
+    index: '0',
```

Comment:
- [ ] Good job implemeting the project requirements, however as mentioned in the requirement we need to make sure 
the keys in each object have the proper datatypes, 

- the description key should have _**string**_ data type, 
- completed key should be a _**boolean**_ either true or false and 
- the index key suppose to store a _**number**_ data type, 

_**description [string].
completed [bool].
index: [number].**_

currently all your keys have string data type. please fix this issue for all your task object's keys